app/src/main/java/me/ccrama/redditslide/Views/CreateCardView.java

```diff
@@ -413,19 +413,27 @@ public boolean onLongClick(View v) {
             RelativeLayout.LayoutParams layoutParams = (RelativeLayout.LayoutParams) v.findViewById(R.id.inside).getLayoutParams();
 
             if (!SettingValues.actionbarVisible && !SettingValues.actionbarTap) {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+                    picParams.addRule(RelativeLayout.START_OF, R.id.secondMenu);
```

Comment:
These bits are the only real logical pieces I added to handle both START/END and LEFT/RIGHT. This was the core bug with RTL/LTR support